src/test/java/seedu/address/testutil/PassengerBuilder.java

```diff
@@ -23,9 +25,9 @@
     public static final String DEFAULT_NAME = "Amy Bee";
     public static final String DEFAULT_PHONE = "85355255";
     public static final String DEFAULT_ADDRESS = "123, Jurong West Ave 6, #08-111";
-    public static final String DEFAULT_TRIPDAY = "FRIDAY";
-    public static final String DEFAULT_TRIPTIME = "1800";
-    public static final String DEFAULT_PRICE = "1.69";
+    public static final DayOfWeek DEFAULT_TRIPDAY = DayOfWeek.FRIDAY;
+    public static final LocalTime DEFAULT_TRIPTIME = LocalTime.of(18, 0);
```

Comment:
i believe this should be ok, because its for the purpose of constructing a passenger